herald/src/app/shared/realm.enum.ts

```diff
@@ -1,5 +1,5 @@
 export const enum Realm {
-  Albion,
-  Hibernia,
-  Midgard
+  Albion = 1,
```

Comment:
Looking at how character-profile.service.ts works, I don't think we need these assigned enum values...do we?  I'd rather not have anything outside of character-profile.service.ts tied to what data we expect the character API to provide.